pyup/providers/gitlab.py

```diff
@@ -43,7 +44,7 @@ def _api(self, token):
                 'Got token "{}": format should be wither "apikey" for '
                 'gitlab.com, or "apikey@https://yourgitlab.local"'.format(
                     token))
-        return Gitlab(host, auth)
+        return Gitlab(host, auth, ssl_verify=(not self.ignore_ssl))
```

Comment:
You did better for GitHub provider above when you assigned `not self.ignore_ssl` to `verify`. Not required, but would be good seeing something similar here. :sunglasses: